doc/src/learn/exchange-integration-faq.md

```diff
@@ -0,0 +1,177 @@
+
+### Can I change the amount of an existing stake during the staking period?
+    
+Yes, you can add to or withdraw your stake from a validator. Use the following methods to modify the stake amount:
+
+Use the [`request_add_delegation`](https://github.com/MystenLabs/sui/blob/58229627970a6e9ff558b156c1cb193f246eaf88/crates/sui-framework/docs/sui_system.md#0x2_sui_system_request_add_delegation) and [`request_add_delegation_with_locked_coin`](https://github.com/MystenLabs/sui/blob/58229627970a6e9ff558b156c1cb193f246eaf88/crates/sui-framework/docs/sui_system.md#0x2_sui_system_request_add_delegation_with_locked_coin) methods to add to the staked amount.
+ 
+Use the [`request_withdraw_delegation`](https://github.com/MystenLabs/sui/blob/58229627970a6e9ff558b156c1cb193f246eaf88/crates/sui-framework/docs/sui_system.md#0x2_sui_system_request_withdraw_delegation) method to withdraw all or part of the delegation.
+
+### Can I change the validator while a coin is actively staked?
+    
+Yes, use the request_switch_delegation method to change the validator while a coin is staked. Examples coming soon.
+    
+### Does Sui require a bonding / warm-up period?
+    
+Yes, but the specifics are still under consideration. We expect a period of only a few days maximum.
+
+### Does Sui require an un-bonding period?
+    
+The current un-bonding period is one week, this may change prior to Mainnet launch.
+
+### Are staking rewards auto-compounded?
+    
+Yes, Sui uses a staking pool approach inspired by liquidity pools. Rewards are added to the pool and auto-compounded through the appreciation of pool token value relative to SUI tokens. 
+
+### Do rewards appear as inbound / outbound on-chain transaction?
+    
+Yes, rewards are added to the staking pool through a special system transaction at epoch boundaries.
+
+### How long does it take to get the first reward after staking? How frequently are rewards paid out?
+    
+Rewards are compounded every epoch, and paid out when you withdraw your stake. You must stake for the entire duration of an epoch to receive rewards for that epoch.
+
+### Is there a minimum and maximum staking amount (for validation and delegation)?
+    
+There will be a minimum amount required and a maximum amount allowed, as well as limits on stake changes within an epoch. 
+    
+ * Validation: Requires a high minimum amount of SUI.
+ * Delegation: Minimum planned to be 1 SUI (subject to change). 
+    
+Specific amounts will be provided when available.
+
+### How does slashing work, and what are the penalties?
+    
+There will not be slashing for the principal stake allocated. Instead, validators will get penalized by having fewer future rewards when these get paid out. Rewards that have already been accrued are not at risk.
+
+### Does Sui support on-chain governance or voting?
+    
+On-chain governance is not implemented for Sui. This is not planned for in the near future.
+
+## Where can I find block details?
+
```

Comment:
Answers to the following questions address specific details around surfacing block details. ?